extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -1025,6 +1030,23 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 								const selectedBlobContainer = this.migrationStateModel._blobContainers.find(blob => blob.name === value);
 								if (selectedBlobContainer && !blobContainerErrorStrings.includes(value)) {
 									this.migrationStateModel._databaseBackup.blobs[index].blobContainer = selectedBlobContainer;
+
+									if (this.migrationStateModel.isSqlVmTarget && utils.isTargetSqlVm2014OrBelow(this.migrationStateModel._targetServerInstance as SqlVMServer)) {
```

Comment:
seems like it is, but just want to confirm that this is being done for both Online and Offline scenarios